### DISCUSSION TITLE
Ideas for Adminshop and gaining stars

### DISCUSSION THREAD

**Discussion by ReiTheGoat01** [Stardust-Labs-MC/StardustMC#26] [Ideas]
When we switch over to 1.20.2, we could use archeology and give sus sand/gravel a rare chance to drop "treasures" these would just be custom items with textures with the only use is to be sold for stars, it would encourage archeology and be a neat way for players to earn stars.

Selling sherds since they are non-farmable possibly for 5 stars.

Increasing Enchanted Golden Apples to 60 from 40 given how much rarer they are compared to heart of the seas.

When the end opens up the selling of dragon heads for 10-15 stars

Selling pigstep and otherside for 100 as unlike the other music discs they can't be farmed, and be able to sell the 5 music disc for those who combined the disc fragments before they were sellable.

**Comment by aisukuma** [comment]
- [ ] Modify loot table for suspicious sand/gravel to drop "treasure"
- [x] Have the Admin Shop buy shards/sherds in the Admin Shop for a certain price
- [x] Increase sell price of Golden Enchanted Apples in the Admin Shop
- [ ] Have the Admin Shop buy non-farmable music discs at the Admin Shop

**Reply by aisukuma** [reply]
Once the server updates to 1.20(.2), I'll have the Admin Shop be able to buy the sherds/shards.

**Comment by aisukuma** [comment]
Sorry, I have not been able to get enough time to work on this, but loot from suspicious sand/gravel will come at a later date
